modules/manage/partials/tiered-storage.adoc

```diff
@@ -869,29 +869,61 @@ rpk topic alter-config <topic_name> --set redpanda.remote.read=true
 
 See also: xref:manage:kubernetes/k-remote-read-replicas.adoc[Remote Read Replicas]
 
-=== Caching
+== Caching
+
+When a consumer fetches an offset range that isn't available locally in the Redpanda data directory, Redpanda downloads remote segments from object storage. These downloaded segments are stored in the object storage cache.
+
+[[max-cache]]
+=== Set a maximum cache size
+
+To ensure that the cache does not grow uncontrollably, which could lead to performance issues or disk space exhaustion, you can control the maximum size of the cache.
+
+Redpanda checks the cache periodically according to the interval set in config_ref:cloud_storage_cache_check_interval,true,tunable-properties[]. If the size of the stored data exceeds the configured limit, the eviction process starts. This process removes segments that haven't been accessed recently until the cache size drops to the target level.
+
+*Related properties*:
+
+- config_ref:cloud_storage_cache_size_percent,true,cluster-properties[]
+- config_ref:cloud_storage_cache_max_objects,true,cluster-properties[]
```

Comment:
The link to this property didn't work for me.

Reply:
Fixed 😄